device/core/src/client_diagnostic.ts

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+import { Message, errors } from 'azure-iot-common';
+
+/**
+ * Data structure to store diagnostic property data
+ */
+export class DiagnosticPropertyData {
```

Comment:
>DiagnosticPropertyData [](start = 13, length = 22)

"PropertyData" doesn't add value to the name

Reply:
fixed